Automattic/jetpack
# VideoPress: fix dropdown menu position

~This PR applies the border color green --jp-green-40 #069E08 to the button when it's focused.~
Also, it fixes the position of the dropdown menu.

Fixes https://github.com/Automattic/jetpack/issues/25827

#### Changes proposed in this Pull Request:

* VideoPress: tweak Edit button focus styles of the VideoDetailsCard component

#### Other information:


#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Test with storybook
* Select the VideoDetailsCard component

* Confirm the dropdown menu position is correct

before | after 
------|------
<img width="384" alt="image" src="https://user-images.githubusercontent.com/77539/186932166-dc6db753-fc54-4724-b819-d6a943e34a4f.png"> | <img width="374" alt="image" src="https://user-images.githubusercontent.com/77539/186932204-355f7c24-de6f-4f68-ab5a-c1aa20f3ab65.png">
